.github/workflows/pull-compliance.yml

```diff
@@ -26,6 +26,19 @@ jobs:
         env:
           TAGS: bindata sqlite sqlite_unlock_notify
 
+  lint-templates:
+    if: needs.files-changed.outputs.templates == 'true'
+    needs: files-changed
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+      - run: pipx install poetry
```

Comment:
Since it's a one time installation, I couldn't find the difference from pip and pipx here.

Reply:
npm method just wraps `pip` so it not suitable. I think pipx is fine, but if you really like, I can change this to `pip` for the actions run.

Reply:
Now using `pip -U`